tests/functional/i/implicit/implicit_str_concat_multiline.py

```diff
@@ -2,3 +2,10 @@
 
 TEST_TUPLE = ('a', 'b'  # [implicit-str-concat]
               'c')
+
+print(
+    "Lorem ipsum dolor sit amet, consectetur adipiscing elit," # [implicit-str-concat]
```

Comment:
Why is this bad?

Reply:
In this file the [``check-str-concat-over-line-jump``](https://github.com/PyCQA/pylint/blob/main/pylintrc#L556) option is set to yes.

Reply:
The default option is no, the other file shows it's not raised by default see https://github.com/PyCQA/pylint/pull/6443/files#diff-2248542e115bc4d2f482c95ab3aba32f207108b89d18abede5c62e18db5e553cR36